### ISSUE TITLE
How we can send events to MSK kafka or ES

### ISSUE THREAD

**Issue by jainpratik163** [heptio/eventrouter#129]
hi,
how we can send events to MSK Kafka or ES as we don't see any documentation or example.
if we have something please share it.

**Comment by aantn** [comment]
@jainpratik163 I don't know if you're still looking for a solution, but you can send k8s events to Kafka with a different open source project I wrote: https://github.com/robusta-dev/robusta

Re. Elasticsearch, we don't support that out of the box, but if you open an issue I'm happy to find a quick solution

**Comment by jainpratik163** [comment]
@aantn yes still looking for a solution.
and this is issue i open for elastic search support.
please let me know if I want to open a new one.

**Comment by aantn** [comment]
@jainpratik163 yes, can you open an issue for robusta? I have a few questions regarding the implementation and it would be best to discuss there
